src/som/interpreter/actors/EventualSendNode.java

```diff
@@ -133,6 +134,23 @@ protected SendNode(final SSymbol selector, final WrapReferenceNode[] wrapArgs,
     }
 
     /**
+     * Update messageReceiverBreakpoint in case that a step into has been made.
+     */
+    private void checkSteppingInto() {
+      this.messageReceiverBreakpoint = insert(Breakpoints.createReceiver(this.source));
```

Comment:
This node seems to be created every time there is a STEP_INTO, is this really necessary? Isn't it sufficient to create it once only?